src/sagas/bucket.js

```diff
@@ -50,11 +50,11 @@ export function* updateBucket(getState, action) {
 }
 
 export function* deleteBucket(getState, action) {
-  const { bid } = action;
+  const { bid, last_modified } = action;
```

Comment:
Scratch it, we're speaking deletion here, so we need the last_modified being passed in the action. This comment still actually applies for updates though.